book/website/collaboration/remote-collab/remote-collab-code-review.md

```diff
@@ -0,0 +1,34 @@
+# Collaborative Code Review
+
+## 1. Problem statement
```

Comment:
I think this section would benefit from clearer language around problem statements and how collaborative code review addresses them, e.g. "There are many reasons why you may want to organise a collaborative code review: 

- to ensure your code is accessible and reproducible
- to get feedback on your code's interface to make it more useable
- to better understand how to solve a particular problem in a new coding language 
- and more...."